searx/webapp.py

```diff
@@ -424,7 +408,7 @@ def render(template_name, override_theme=None, **kwargs):
 
     kwargs['results_on_new_tab'] = request.preferences.get_value('results_on_new_tab')
 
-    kwargs['unicode'] = unicode
```

Comment:
The unicode function can be removed.
The function is used only the 404.html pages.